src/generator.rs

```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 Helsing GmbH. All rights reserved.
```

Comment:
I don't think this is the right header for a project that's licensed under Apache 2.0. W/ apache 2.0, all files need an apache 2 header, see manual https://github.com/helsing-ai/buffrs/blob/main/LICENSE#L179

Reply:
Good flag. I copied this from another file but I agree it doesn't seem right. Will fix tomorrow and we should probably follow up with a fix to other files too.